.github/workflows/test.yaml

```
@@ -0,0 +1,31 @@
+name: Run Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '20'  
+
+    - name: Install dependencies
+      run: npm install
```

```suggestion
      run: npm ci
```
We can use `ci` to enforce the versions specified in the lock file.